tests/test_incremental.py

```diff
@@ -96,22 +99,20 @@ def test_scoring_string(scheduler, xy_classification, scoring):
         assert callable(check_scoring(clf, scoring=scoring))
         clf.fit(X, y, classes=np.unique(y))
         clf.score(X, y)
-        clf.estimator.score(X, y)
 
 
 def test_fit_ndarrays():
     X = np.ones((10, 5))
-    y = np.ones(10)
+    y = np.concatenate([np.zeros(5), np.ones(5)])
```

Comment:
Needed a mix of zeros and ones. I'm not entirely sure what changed to necessitate that...